### PR TITLE
Documentation Standard Updates

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
This pull request was created automatically 🎉:

- [x] Ensure `.github/` files have been properly updated (templates are updated and auto releaser is included)
- [x] Confirm all appropriate end models are included in the `.quickstart/quickstart.yml` file as `public_models` (only for transform repos)
- Verify the README has made all necessary updates:
    - [x] Removal of emojis
    - [x] Replacing exclamation points with periods in sentences
    - [x] Adjusting all headers to ensure only one H1 heading
    - [x] Confirming a section header is included for Quickstart UI rendering (only for transform repos)
    - [x] Renaming `**models` to `**tables**` (only for transform repos)
    - [x] Removal of Fivetran community and office hour resources (if necessary)

### CODE REVIEW

**Review by jlmendgom5TRAN — APPROVED** [review]
Made some edits. Looks good to me!

**Review by fivetran-catfritz — COMMENTED** [review]
@fivetran-joemarkiewicz One quick question before I approve, one the be checkboxes say `Renaming **models to **tables** (only for transform repos)`. Is that meant that all instances of "models" should be "tables" or just in the table with the table and its description? For example, [L22](https://github.com/fivetran/dbt_apple_search_ads/blob/40b50a3cfeea2963e31cb3d8d5fb164c91a31cf8/README.md?plain=1#L22) calls them models instead of tables.

File: README.md [hunk 1]
```diff
@@ -11,16 +11,17 @@
...
 - Enables you to better understand the performance of your ads across varying grains:
-  - Providing an organization, campaign, ad group, keyword, search term and ad level reports.
+    - Providing an organization, campaign, ad group, keyword, search term and ad level reports.
 - Materializes output models designed to work simultaneously with our [multi-platform Ad Reporting package](https://github.com/fivetran/dbt_ad_reporting).
 - Generates a comprehensive data dictionary of your source and modeled Apple Search Ads data through the [dbt docs site](https://fivetran.github.io/dbt_apple_search_ads/).
 
-The following table provides a detailed list of all models materialized within this package by default. 
+<!--section="apple_search_ads_transformation_model"-->
+The following table provides a detailed list of all models materialized within this package by default.
```
Comment (fivetran-joemarkiewicz) [inline README.md:22]:
```suggestion
The following table provides a detailed list of all tables materialized within this package by default.
```

**Comment by fivetran-joemarkiewicz** [comment]
@fivetran-catfritz great callout. We will want to make sure the wording rendered in the Quickstart UI is referring to "tables". So in your callout we will want to make that update. But we can leave the other non UI references as models.

I just made the update. Let me know if you have other questions.

**Review by fivetran-catfritz — APPROVED** [review]
@fivetran-joemarkiewicz thanks for the explanation. lgtm!

**Pull request merged**